lib/virt_autotest/utils.pm

```diff
@@ -440,14 +440,17 @@ sub check_guest_health {
         if (script_run("ssh root\@$vm 'ping -c3 8.8.8.8'") == 0) {
             record_info("Healthy guest!", "$vm looks good so far!");
         } else {
-            record_info("Possible network inaccessibility", "Unable to access outside network from $vm!", result => 'fail');
-            $failures .= " Unable to access outside network from $vm!";
+            my $fail_msg = "Unable to access outside network from $vm";
+            record_info("Possible network inaccessibility", "$fail_msg!", result => 'fail');
+            record_soft_failure("$fail_msg, please track it in poo#192382");
```

Comment:
Which works best for reviewers? Use poo#192382 to track all such soft-fail issues(investigation and solution) , or ask reviewers to create subtasks under it to track each specific issue, or just use it as reminder ticket and ask reviewer to handle with regular process(create separate ticket anywhere suitable)?

Reply:
You mean using this ticket to track all failures in check_guest_health function ? Then how about specific issues like https://progress.opensuse.org/issues/191431?

For me, as long as it could warn all reviewers to open the soft-fail jobs, check/investigate(and even fix if needed) the red box failures, it will be fine.

Let's also involve  MU reviewers for opinion. @tbaev @martinsmarcelo @RoyCai7 How would you prefer? See  poo#192382 for more background info and expectation.

Reply:
I don't bother how the failures are resolved, by creating a sub ticket, posting comments in this root ticket or even creating a separate ticket. I focus more on the results that the failure is expected to be fix eventually. Up to you and team.

Reply:
Track issues in poo#192382 and create sub tickets if suitable is okay with me, I like Roy's proposal too.

Reply:
Roy, you mean only installing the browser addon  Tampermonkey, it is enough?

Reply:
yes. I think so.

Reply:
Good, then let's use that ticket to trace this part for each reviewers' browser setup.